subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -418,6 +418,10 @@ uint8_t ll_adv_params_set(uint16_t interval, uint8_t adv_type,
 			pdu->tx_addr = 0;
 		}
 
+		/* TargetA flag */
+		if (pri_hdr_prev.tgt_addr) {
+			pri_dptr_prev += BDADDR_SIZE;
```

Comment:
```suggestion
			pri_dptr_prev += TARGETA_SIZE;
```

Reply:
I see the others usually use a `sizeof` of some struct; we don't have a fitting struct for this one?

Reply:
No, no struct is defined in Controller, address type and address has been scattered in the PDUs, and using a struct adds overheads and there is no strong reasons to only use it for `sizeof` to return a value 6.

No action here, as TARGETA_SIZE is only used in the context of calculating legacy PDU size, and all other instances use `BDADDR_SIZE`.

Reply:
> and using a struct adds overheads and there is no strong reasons to only use it for sizeof to return a value 6.

`sizeof` would end up being a compile-time constant, would it not? And should have no overhead. I agree that there is no strong reasons to use it but for the sake of consistency (e.g. why not just use a #define for all the other cases, instead of a sizeof of a struct?). 

> No action here, as TARGETA_SIZE is only used in the context of calculating legacy PDU size

Just because it hasn't been used for anything before, is not a particularly good reason to not use it now. Semantically, since this deals with the Target Address, using a macro related to the size of the target address makes more sense to me. 

> and all other instances use BDADDR_SIZE.

Some of them should arguably also have used some of the other #defines.